html_app/model/Treatment.js

```diff
@@ -35,85 +35,91 @@ scb.Treatment = function scb_Treatment(data, context, parent) {
 	scb.Utils.initialize_accessor_field(self, data, 'facs', {}, null, context);
 
 	// should be getter only
-	scb.Utils.initialize_accessor_field(self, data, 'schedule_value', "0", null, context);
+	scb.Utils.initialize_accessor_field(self, data, 'start_time_value', "0", null, context);
 	// should be getter only
 	scb.Utils.initialize_accessor_field(self, data, 'duration_value', "0", null, context);
+    if(data.start_time_value !== "0"){
+        scb.Utils.initialize_field(data, 'start_time', self.schedule_value);
+        Object.defineProperty(self, 'start_time', {
+            get : function() {
+                var time = parseFloat(data.schedule_value);
```

Comment:
This is the old code that is not used by current assignments. I think we will be deleting it soon.